bindings/nodejs/lib/types/utils/type_aliases.ts

```diff
@@ -2,3 +2,4 @@
 // SPDX-License-Identifier: Apache-2.0
 
 export type u16 = number;
+export type u64 = bigint;
```

Comment:
AFAIK, `Number.MAX_SAFE_INTEGER` < `u64::MAX`  so `u64`  should be `bigint` in JS.

Reply:
yea checks out

Reply:
Can we create an issue then?

Reply:
https://github.com/iotaledger/iota-sdk/issues/966 added here